src/test/js/self_registration_test.js

```diff
@@ -77,7 +77,6 @@ Scenario('@functional @selfregister User Validation errors', async ({ I }) => {
     I.see('Your last name is invalid');
     I.see('Last name has to be longer than 1 character and should not include digits nor any of these characters:')
     I.see('Sign in to your account.');
-    await I.runAccessibilityTest();
```

Comment:
Does removing those lines reduce the accessibility test coverage?

Reply:
No Kremena, most of these tests are covered in different scenarios.